libwallet/src/api_impl/owner.rs

```diff
@@ -75,6 +77,29 @@ where
 	w.set_parent_key_id_by_name(label)
 }
 
+/// Retrieve the slatepack address for the current parent key at
+/// the given index
+pub fn get_rewind_hash<'a, L, C, K>(
```

Comment:
I'm wondering whether we should reuse the code from here https://github.com/mimblewimble/grin/blob/master/keychain/src/view_key.rs#L97-L100 to avoid having duplicate computations like this (it's not an issue now, but if/when someone updates one part of the code, it would need to guess that it also exists in this other repo).

Reply:
Agree. Let's do it directly then. I PR this on the node code https://github.com/mimblewimble/grin/pull/3674 in order to use it there.

Reply:
Error